app/lib/features/tasks/pages/task_item_detail_page.dart

```diff
@@ -353,11 +356,7 @@ class TaskItemDetailPage extends ConsumerWidget {
     return ListTile(
       dense: true,
       leading: const Icon(Atlas.business_man_thin),
-      title: Text(
-        L10n.of(context).assignment,
-        style: Theme.of(context).textTheme.bodyMedium,
-      ),
-      subtitle: task.isAssignedToMe()
+      title: task.isAssignedToMe()
```

Comment:
Something I might have overlooked before but whey do we not show the assigned if it isn't the user themselves? This check seems wrong...

Reply:
Ahh I see,
I have did required changes in the task details page as below.

<img width="370" alt="Screenshot 2024-06-25 at 10 09 49 AM" src="https://github.com/acterglobal/a3/assets/51526480/88d65659-d83a-472a-bf55-b84bac302944">

Reply:
How ever I find some additional issue on assignment which I reported here.
https://github.com/acterglobal/a3/issues/1877